docs/vedlegg-a-utdypende-beskrivelse.adoc

```diff
@@ -125,16 +125,16 @@ For å sørge for at deltakerne bruker identiske begreper.
 
 4. Deltakerne utformer definisjoner og fastsette termer for nye, felles begreper.
 
-5. Deltakerne registrerer og tilgjengeliggjør begrepene som virksomhetene blir enige om å harmonisere i en termbase, f.eks. https://fellesdatakatalog.digdir.no/concepts[Felles begrepskatalog] eller https://oda.uib.no/app?ma=term&mg=aterm[Termportalen].
+5. Deltakerne registrerer og tilgjengeliggjør begrepene som virksomhetene blir enige om å harmonisere i en termbase, f.eks. https://fellesdatakatalog.digdir.no/concepts[begrepskatalogen] eller https://www.uib.no/ub/fagressurser/spesialsamlingene/121707/termportalen[Termportalen].
```

Comment:
```suggestion
5. Deltakerne registrerer og tilgjengeliggjør begrepene som virksomhetene blir enige om å harmonisere i en termbase, f.eks. https://fellesdatakatalog.digdir.no/concepts[Begrepskatalogen] eller https://www.uib.no/ub/fagressurser/spesialsamlingene/121707/termportalen[Termportalen].
```

Reply:
Språkrådet (som leder dette arbeidet) mener vi ikke bør bruke stor forbokstav her. Man ser hverken Begrepskatalog eller Felles begrepskatalog når man er i "katalogportalen" vår. Jeg velger derfor å la det være liten forbokstav i denne omgangen, og skal be "katalogteamet" vurdere denne problemstillingen (Er det flere kataloger i katalogen? I så fall, hva er hva og hva heter de?).